tests/kernel/kernel_attributes_vec_hint.cpp

```diff
@@ -0,0 +1,212 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Copyright (c) 2023 The Khronos Group Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+*******************************************************************************/
+
+#include "../common/common.h"
+#include "../common/disabled_for_test_case.h"
+#include "kernel_attributes.h"
+
+using namespace kernel_attributes;
+
+// FIXME: enable when [[sycl::vec_type_hint(<type>)]] is implemented
+// https://github.com/intel/llvm/issues/9836
+#ifndef SYCL_CTS_COMPILING_WITH_DPCPP
```

Comment:
Shouldn't the test compile even if the impl doesn't support the attribute? Unknown attributes should be just ignored by the C++ compiler.

Reply:
I have compilation failures with attributes shown in issue above

Reply:
Interesting, definitely seems like a DPC++ bug then.